proxy/coco/tests/gossip.rs

```diff
@@ -174,9 +177,24 @@ async fn can_ask_and_clone_project() -> Result<(), Box<dyn std::error::Error>> {
 
     requested(query_listener, &urn).await?;
     assert_cloned(clone_listener, &urn.clone(), alice_peer_id).await?;
-    // TODO(finto): List projects
-    let project = state::get_project(&bob_peer, urn).await;
+    let project = state::get_project(&bob_peer, urn.clone()).await;
     assert!(project.is_ok());
 
+    let announced = async_stream::stream! { loop { yield alice_events.recv().await } }
+        .filter_map(|res| match res.unwrap() {
+            coco::PeerEvent::GossipFetched {
+                gossip, provider, ..
+            } if provider.peer_id == bob_peer_id && gossip.urn.id == urn.id => {
```

Comment:
So in my other comment, I noticed we weren't using the local `PeerId`. I have there are two `GossipFetched` events appearing, one with `alice`'s `PeerId` and one with `bob`'s. But I'm not 100% on this :thinking: Nonetheless, I'm a little surprised this worked :smile: